src/pybind/mgr/dashboard/controllers/mgr_modules.py

```diff
@@ -194,3 +194,11 @@ def _convert_module_options(self, options):
                     elif option[name]:  # Skip empty entries
                         option[name] = cls(option[name])
         return options
+
+
+@UIRouter('/mgr/module', Scope.USER)
+@APIDoc("Dashboard UI helper functions; user scope", "MgrModule")
+class MgrModulesUI(MgrModules):
+
+    def get(self, module_name):
+        return MgrModules.get(self, module_name)
```

Comment:
why this has a `Scope.USER`? Probably its not good to expose all the mgr config with a different scope since this is admin level details. We should have an endpoint for dashboard (probably in the health.py). rather than having multiple smaller endpoint, maybe a monolithic endpoint to get all the minimal things? wdyt?

Reply:
>why this has a Scope.USER? 

I thought this parameter was required so I went with the most fitting value (IMO), just realised this is optional. So will remove the `Scope.USER`

>Probably its not good to expose all the mgr config with a different scope since this is admin level details.

What do you mean with exposed? The point of `@UIRouter` was that is not being exposed, right?
If we want to show info for like the telemetry module, to get if the module is activated you need admin access, so we will be accessing admin level info anyways, so I thought this would do the trick.

I suppose we can have a new Dashboard endpoint to put all this (with the minimal info) in the `health.py`